### PR TITLE
Comments/edits from our 3GPP experts

### PR DESCRIPTION
**Pull request by mirjak**

### CODE REVIEW

File: draft-bonaventure-quic-atsss-overview.mkd [hunk 1]
```diff
@@ -232,9 +232,10 @@ to improve the up and download throughput performance of individual flows.
 The GiGA LTE and GiGA 5G services deployed using Multipath TCP in South Korea are examples of this use case {{IETFJ16}}.
 
 To support these different use cases in 5G networks, 3GPP is defining the
-Access Traffic Steering, Switching and Splitting (ATSSS) service {{TS23501}}. This work is
+Access Traffic Steering, Switching and Splitting (ATSSS) service {{TS23501}}
+to allow the simultaneous use of the combination of a 3GPP and a non-3GPP network interface. This work is
 further adopted by the Broadband Forum to provide similar capabilities to residential gateways
-equipped with multiple access interfaces, in the continuity of the Hybrid Access Networks {{TR-348}}.
+equipped with multiple access interfaces, in the continuity of the Hybrid Access Networks {{TR-348}}.
```
Comment (mpiraux) [inline draft-bonaventure-quic-atsss-overview.mkd:238]:
```suggestion
equipped with multiple access interfaces, in the continuity of the Hybrid Access Networks {{TR-348}}.
```

File: draft-bonaventure-quic-atsss-overview.mkd [hunk 2]
```diff
@@ -232,9 +232,10 @@ to improve the up and download throughput performance of individual flows.
 The GiGA LTE and GiGA 5G services deployed using Multipath TCP in South Korea are examples of this use case {{IETFJ16}}.
 
 To support these different use cases in 5G networks, 3GPP is defining the
-Access Traffic Steering, Switching and Splitting (ATSSS) service {{TS23501}}. This work is
+Access Traffic Steering, Switching and Splitting (ATSSS) service {{TS23501}}
+to allow the simultaneous use of the combination of a 3GPP and a non-3GPP network interface. This work is
```
Comment (mpiraux) [inline draft-bonaventure-quic-atsss-overview.mkd:236]: I would rephrase this as either "simultaneous use" or "combination of", but I feel the two together are redundant.

File: draft-bonaventure-quic-atsss-overview.mkd [hunk 3]
```diff
@@ -480,7 +481,7 @@ The MPTCP-based ATSSS approach discussed in the previous section is specific to
 obviously. Therefore it does not support non-TCP traffic, such as
 UDP, QUIC {{QUIC-Deployment}} or IPsec and Datagram Transport Layer Security (DTLS) Virtual Private Network (VPN) services.
 
-As the share of these protocols grows, mainly driven by QUIC deployment, a future ATSSS needs to extend beyond supporting TCP only. The seamless handover provided by ATSSS is particularly useful for real-time traffic (e.g., voice or video calls), 
+As the share of these protocols grows, mainly driven by QUIC deployment, a future ATSSS needs to extend beyond supporting TCP only. There is a funnctin called ATSSS-LL that operate on a low layer and can therefore also be applied to non-TCP traffic but this services does not support traffic splitting but only steering and expectiionally switching. The seamless handover provided by splitting is particularly useful for real-time traffic (e.g., voice or video calls),
```
Comment (mpiraux) [inline draft-bonaventure-quic-atsss-overview.mkd:484]:
```suggestion
As the share of these protocols grows, mainly driven by QUIC deployment, a future ATSSS needs to extend beyond supporting TCP only. An existing function called ATSSS-LL operates on a lower layer and can therefore also be applied to non-TCP traffic, but it only supports traffic steering and (exceptionally) traffic switching but not traffic splitting. The seamless handover provided by traffic splitting is particularly useful for real-time traffic (e.g., voice or video calls), 
```

File: draft-bonaventure-quic-atsss-overview.mkd [hunk 4]
```diff
@@ -490,12 +491,12 @@ The work within 3GPP now focuses on using QUIC as the baseline for ATSSS Phase 2
 ### QUIC and Multihoming
 
 
-For non-TCP traffic, QUIC is already the dominant part of UDP traffic. A solution as realized with the Convert Protocol for (MP)TCP is not possible for QUIC as a QUIC connection cannot be intercepted and converted as in the ATSSS architecture for (MP)TCP ({{fig-mptcp-arch}}). For (MP)TCP only the transport protocol (TCP) is intercepted, while transport security provided by Transport Layer Security (TLS) on top of TCP stays in tact. For QUIC transport, security is integrated into the transport protocol and thus cannot be intercepted.
+For non-TCP traffic, QUIC is already the dominant part of UDP traffic. A solution as realized with the Convert Protocol for (MP)TCP is not possible for QUIC as a QUIC connection cannot be intercepted and converted as in the ATSSS architecture for (MP)TCP ({{fig-mptcp-arch}}). For (MP)TCP only the transport protocol (TCP) is intercepted, while transport security provided by Transport Layer Security (TLS) on top of TCP stays intact. For QUIC transport, security is integrated into the transport protocol and thus cannot be intercepted.
```
Comment (mpiraux) [inline draft-bonaventure-quic-atsss-overview.mkd:494]:
```suggestion
For non-TCP traffic, QUIC is already the dominant part of UDP traffic. A solution as realized with the Convert Protocol for (MP)TCP is not possible for QUIC as a QUIC connection cannot be intercepted and converted as in the ATSSS architecture for (MP)TCP ({{fig-mptcp-arch}}). For (MP)TCP, only the transport protocol (TCP) is intercepted, while transport security provided by Transport Layer Security (TLS) on top of TCP stays intact. For QUIC transport, security is integrated into the transport protocol and thus cannot be intercepted.
```

Comment (mpiraux) [reply draft-bonaventure-quic-atsss-overview.mkd:494]: I added a comma after "For (MP)TCP" for readability.

File: draft-bonaventure-quic-atsss-overview.mkd [hunk 5]
```diff
@@ -563,14 +564,14 @@ but splitting cannot be supported.
 Path validation
 induces a delay when switching as packets are buffered. Further, congestion control state is
 reset on a new path and needs to ramp up. When frequent handovers happen, 
-splitting traffic over multiple paths simulaneously can be beneficial for
+splitting traffic over two paths simulaneously can be beneficial for
```
Comment (mpiraux) [inline draft-bonaventure-quic-atsss-overview.mkd:567]: I haven't read the full document in a while. If you feel this is editoral, then we should go ahead and change that. If it's a design choice (i.e. restricting to two paths instead of more), then it might be good to add a sentence stating the presented approach could be extended to more than two paths.

Comment (SpencerDawkins) [reply draft-bonaventure-quic-atsss-overview.mkd:567]: I agree with Maxime - ATSSS as practiced in Rel 16, and apparently Rel 17, is closely tied to current UE capabilities, so, two paths. ISTM that we would want to point out that more than two paths could pop up at any time, and ISTM unlikely that the QUIC working group would be interested in preserving the two-path limitation in work that they do.

**Comment by SpencerDawkins** [comment]
Mirja, just checking - are the 3GPP experts already thanked in the Acknowledgements section?
